_shared_content/ECS parsing standariziation for EDR.md

```diff
@@ -0,0 +1,408 @@
+        "integrity_level": "High",
+        "sigma_rule_content": "title: Local User Created via PowerShellw",
+        "ppid": 4364,
+        "process_unique_id": "93086cd1-8271-4941-802a-0039d517a425",
+        "hashes": {
+          "md5": "9d8e30eff21108092d5980c931876b7e",
+     
+        "session": 2,
+        "log_platform_flag": 0,
+        "grandparent_unique_id": "93086cd1-8271-4941-0429-0029050971a0",
+        "dont_create_process": true,
+        "parent_commandline": "C:\\Users\\administrateur\\AppData\\Local\\Microsoft\\WindowsApps\\python3.exe .\\normal-traffic.py --activities perform_user_management",
+        "process_name": "powershell.exe",
+        "username": "vm-windows\\administrateur",
+        "fake_parent_commandline": "",
+        "system_event_type": "powershell_event",
+        "pe_timestamp_int": 144346722,
+        "size": 450560,
+        "grandparent_commandline": "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe -noexit -command try { . \"c:\\Users\\administrateur\\AppData\\Local\\Programs\\Microsoft VS Code\\resources\\app\\out\\vs\\workbench\\contrib\\terminal\\browser\\media\\shellIntegration.ps1\" } catch {}",
+        "usersid": "S-1-5-21-3860028710-1894839758-596913598-500",
+        "image_name": "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe",
+        "commandline": "powershell.exe -Command New-LocalUser -Name User9549 -Password (ConvertTo-SecureString -AsPlainText \"Password123\" -Force)",
+        "parent_image": "C:\\Program Files\\WindowsApps\\PythonSoftwareFoundation.Python.3.12_3.12.1264.0_x64__qbz5n2kfra8p0\\python3.12.exe",
+        "fake_ppid": 0,
+        "pe_timestamp": "1974-07-29T16:18:42.000Z",
+        "create_time": "2024-07-12T08:13:13.221Z",
+        "signed": true,
+        "ioc_matches": [],
+        "log_type": "process"
+      },
+      "agent": {
+        "version": "3.10.2",
+        "hostname": "vm-windows",
+        "ostype": "windows",
+        "osversion": "10.0.22631",
+        "distroid": null,
+        "domain": null,
+        "agentid": "1553619e-8271-4941-8e10-9153e99cc5dc",
+        "dnsdomainname": null,
+        "osproducttype": "Windows 11 Enterprise",
+        "domainname": "WORKGROUP",
+        "additional_info": {}
+      },
+      "detection_origin": "agent",
+      "rule_name": "Local User Created via PowerShell",
+      "alert_type": "sigma",
+      "log_type": "alert"
+    }
+  ```
+For example, the event characteristics are derived from the log_type and alert_subtype(`process.log_type`) fields in the raw message:
+* The fields `event.kind` and `event.dataset` are set to alert because the log_type in the raw event is alert.
+* The field `event.category` is set to 'process' and `event.type` is set to 'start' because the `process.log_type` field of the raw event is identified as process.
+
+**ECS Fields After Parsing**
+    
+```json
```

Comment:
```suggestion
  ```
For example, the event characteristics are derived from the log_type and alert_subtype(`process.log_type`) fields in the raw message:
* The fields `event.kind` and `event.dataset` are set to alert because the log_type in the raw event is alert.
* The field `event.category` is set to 'process' and `event.type` is set to 'start' because the `process.log_type` field of the raw event is identified as process.

**ECS Fields After Parsing**
    
```json
```